.github/workflows/alpha.yml

```diff
@@ -77,6 +77,14 @@ jobs:
           COSIGN_PWD: ${{ secrets.COSIGN_PWD }}
           GORELEASER_CURRENT_TAG: v0.0.0-alpha
 
+      - name: Delete outdate release
+        uses: dev-drprasad/delete-tag-and-release@v0.1.2
```

Comment:
I'm afraid of using this action. Would it be too much work to do manually?

Reply:
It would be more work and we would have the same result.